docs/Too-Many-Constants.md

```diff
@@ -0,0 +1,37 @@
+## Introduction
+
+_Too Many Constants_ is a case of [Large Class](Large-Class.md).
+
+## Example
+
+Given this configuration
+
+```yaml
+TooManyConstants:
+  max_constants: 3
+```
+
+and this code:
+
+```Ruby
+class Smelly
+  CONST_1 = :dummy
+  CONST_2 = :dummy
+  CONST_3 = :dummy
+  CONST_4 = :dummy
+end
+```
+
+Reek would emit the following warning:
+
+```
+test.rb -- 5 warnings:
```

Comment:
Lets redact this to say "1", otherwise it is confusing for the reader without seeing all the other  warnings.